App does not start anymore: NoSuchMethodException

When opening the app (either via launcher or widget to a pass) I only see a blank screen and the app crashing. Logcat:

```
2020-03-03 20:39:25.238 5482-5482/? E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.ligi.passandroid, PID: 5482
    java.lang.RuntimeException: Unable to start activity ComponentInfo{org.ligi.passandroid/org.ligi.passandroid.ui.PassListActivity}: java.lang.NoSuchMethodException: parameter type is null
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3270)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3409)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:83)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2016)
        at android.os.Handler.dispatchMessage(Handler.java:107)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7356)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
     Caused by: java.lang.NoSuchMethodException: parameter type is null
        at java.lang.Class.getConstructor0(Class.java:2326)
        at java.lang.Class.getDeclaredConstructor(Class.java:2170)
        at org.ligi.passandroid.model.pass.BarCodeJsonAdapter.fromJson(SourceFile:14)
        at org.ligi.passandroid.model.pass.BarCodeJsonAdapter.fromJson(SourceFile:1)
        at com.squareup.moshi.internal.NullSafeJsonAdapter.fromJson(SourceFile:3)
        at org.ligi.passandroid.model.pass.PassImplJsonAdapter.fromJson(SourceFile:18)
        at org.ligi.passandroid.model.pass.PassImplJsonAdapter.fromJson(SourceFile:1)
        at com.squareup.moshi.internal.NullSafeJsonAdapter.fromJson(SourceFile:3)
        at com.squareup.moshi.JsonAdapter.fromJson(SourceFile:1)
        at org.ligi.passandroid.model.AndroidFileSystemPassStore.readPass(SourceFile:7)
        at org.ligi.passandroid.model.AndroidFileSystemPassStore.getPassbookForId(SourceFile:1)
        at org.ligi.passandroid.model.AndroidFileSystemPassStore.syncPassStoreWithClassifier(SourceFile:4)
        at org.ligi.passandroid.ui.PassListActivity.onCreate(SourceFile:17)
        at android.app.Activity.performCreate(Activity.java:7825)
        at android.app.Activity.performCreate(Activity.java:7814)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1306)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3245)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3409) 
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:83) 
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135) 
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2016) 
        at android.os.Handler.dispatchMessage(Handler.java:107) 
        at android.os.Looper.loop(Looper.java:214) 
        at android.app.ActivityThread.main(ActivityThread.java:7356) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930) 
2020-03-03 20:39:25.613 618-618/? E/Layer: [Surface(name=AppWindowToken{893c34d token=Token{bb062e4 ActivityRecord{299f677 u0 org.ligi.passandroid/.ui.PassListActivity t14757}}})/@0x2ea9681 - animation-leash#0] No local sync point found
2020-03-03 20:39:25.613 618-618/? E/Layer: [Surface(name=AppWindowToken{893c34d token=Token{bb062e4 ActivityRecord{299f677 u0 org.ligi.passandroid/.ui.PassListActivity t14757}}})/@0x2ea9681 - animation-leash#0] No local sync point found
2020-03-03 20:39:29.241 5482-5482/? D/PassAndroid: Writing unhandled exception to: /data/user/0/org.ligi.passandroid/files/3.5.4-1583264369241.tracedroid
2020-03-03 20:39:29.242 5482-5482/? D/PassAndroid: java.lang.RuntimeException: Unable to start activity ComponentInfo{org.ligi.passandroid/org.ligi.passandroid.ui.PassListActivity}: java.lang.NoSuchMethodException: parameter type is null
```

--- a/build.gradle
+++ b/build.gradle
@@ -7,11 +7,13 @@ buildscript {
     repositories {
         jcenter()
         google()
+        maven { url 'https://storage.googleapis.com/r8-releases/raw' }
         maven { url 'https://www.jitpack.io' }
     }
 
     dependencies {
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+        classpath 'com.android.tools:r8:2.0.39' // Must be before the Gradle Plugin for Android.
         classpath 'com.android.tools.build:gradle:3.6.1'
         classpath 'com.github.trevjonez.composer-gradle-plugin:plugin:0.13.1'
         classpath 'com.github.ben-manes:gradle-versions-plugin:0.28.0'

